nisystemlink/clients/assetmanagement/models/_asset_create_request.py

```diff
@@ -0,0 +1,98 @@
+from typing import Dict, List, Optional
+
+from nisystemlink.clients.core._uplink._json_model import JsonModel
+
+from ._asset import (
+    AssetBusType,
+    AssetDiscoveryType,
+    AssetLocation,
+    AssetType,
+    ExternalCalibration,
+    SelfCalibration,
+    TemperatureSensor,
+)
+
+
+class AssetCreateRequest(JsonModel):
+    """Model for an object describing the properties for creating an asset."""
+
+    model_name: Optional[str] = None
```

Comment:
There is a caveat to the model_name being optional. The service requires the asset to be uniquely identifiable by a combination of modelName, modelNumber, serialNumber, vendorName, vendorNumber, busType, and minionId. We do have the ability to write a Pydantic validator that would enforce this in the client. That validation seems like a nice to have, but I don't like the idea of duplicating service validation logic into the client and keeping those two in sync if the service behavior ever needs to change. So all that said, I don't think you need to change anything, just adding the context and opening up for comment. FYI @CiprianAnton